src/renderer/wg_engine/tvgWgRenderData.cpp

```diff
@@ -600,3 +600,65 @@ void WgRenderDataPicturePool::release(WgContext& context)
     mPool.clear();
     mList.clear();
 }
+
+//***********************************************************************
+// WgRenderDataGaussian
+//***********************************************************************
+
+void WgRenderDataGaussian::update(WgContext& context, const RenderEffectGaussianBlur* gaussian, const Matrix& transform)
+{
+    assert(gaussian);
+    // compute gaussian blur data
```

Comment:
please remove all useless comments in this method. "WgRenderDataGaussian::update()" already says it.